python_envs/environment.yaml

```
@@ -0,0 +1,51 @@
+channels:
+- conda-forge
+dependencies:
+- python>=3.12
+- aiohttp
+- cartopy
+- cdsapi
+- cfunits
+- cmocean
+- cudatoolkit
+- dask
+- dask-jobqueue
+- distributed
+- fastparquet
+- hdf5plugin
```

https://xarray.dev/blog/flox
```suggestion
- fastparquet
- flox
- hdf5plugin
```